kickstart/provision.erb

```diff
@@ -183,12 +183,14 @@ echo "updating system time"
 echo 'proxy = <%= proxy_uri %>' >> /etc/yum.conf
 <% end -%>
 
+<% unless @host.param_false?('update_base_packages') -%>
```

Comment:
@domcleal provision/finish.erb hasn't got any of the parameters in the doc section, if this is an omission rather than intentional I can have a look through and a parameter doc block to that file?